src/ixdat/exporters/csv_exporter.py

```diff
@@ -42,16 +52,18 @@ def export_measurement(self, measurement, path_to_file, v_list=None, tspan=None)
             "".join([s_name + self.delim for s_name in s_list])[: -len(self.delim)]
             + "\n"
         )
+        #  TODO: Header may have to indicate which time goes with which value.
 
         lines = [header_line]
         max_length = max([len(data) for data in columns_data.values()])
         for n in range(max_length):
             line = ""
             for s_name in s_list:
                 if len(columns_data[s_name]) > n:
-                    line.join(str(columns_data[s_name][n]) + self.delim)
+                    line = line + str(columns_data[s_name][n]) + self.delim
                 else:
-                    line.join(self.delim)
+                    line = line + self.delim
+            line = line + "\n"
```

Comment:
I guess you could in-place add all those `line`s, to make it a little easier to parse what is added

Reply:
Ah no. I meant in the new lines. I just meant that you could do:
```python
        line += self.delim
line += "\n"
```
instead of
```python
        line = line + self.delim
line = line + "\n"
```
which I think is marginally easier to read.

Reply:
done.